testing/python/metafunc.py

```diff
@@ -1323,25 +1323,29 @@ def test_foo(x):
         reprec = testdir.runpytest()
         reprec.assert_outcomes(passed=4)
 
-    @pytest.mark.parametrize("attr", ["parametrise", "parameterize", "parameterise"])
-    def test_parametrize_misspelling(self, testdir, attr):
+    def test_parametrize_misspelling(self, testdir):
```

Comment:
i believe we can now or as a follow-up turn this into a unittest (and leave the checks for all misspellings)

after all its now a with pytest.raises(pytest.fail.Exception): pytest.mark.parametrise/...

Reply:
FWIW I think it's not worth it: it's good to have it as an integration test in general, but it's enough to test one variant then.